src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -42,6 +42,11 @@ object ConsignmentFields {
         "fileChecks",
         FileChecksType,
         resolve = context => DeferFileChecksProgress(context.value.consignmentid)
+      ),
+      Field(
+        "parentFolder",
+        StringType,
```

Comment:
Should this be `OptionType(StringType)`, since the parent folder name might not exist yet?

Reply:
Ah yes you're right! :eyes: